### DISCUSSION TITLE
Pomysły na dodatkowe "bajery"

### DISCUSSION THREAD

**Discussion by lilareliga** [ktprezes/CodersCamp2020.Project2.JavaScript.StarWarsQuiz#19] [Ideas]
Hej,
Marcin zaproponował kilka bajerów do dodania, jak już nasz projekt będzie gotowy, a nam zostanie czasu i energii ;) Spisuję, poddaję pod dyskusję i zapraszam do komentowania i zgłaszania kolejnych pomysłów. 

1. Dźwięki przy sprawdzaniu odpowiedzi poprawna / niepoprawna.
2. Po zatwierdzeniu odpowiedzi (wybraniu jeśli nie ma potwierdzenia) wyłączenie przycisków odpowiedzi
3. Animacja: pytanie wyświetla się literka po literce (zamiast cały blok na raz)
4. Wstępny widok z opisem w formacie intro star warsów, z ta perspektywa i przesuwanym tekstem
5. W podsumowaniu wyników w game over checkbox z możliwością filtrowania np. pokaż tylko błędne/prawidłowe odpowiedzi

**Comment by 8bcb** [comment]
Ad. 1 wprowadźmy to, fajne - zagadam do KLIENTA
Ad. 2 chyba nie rozumiem, ktoś rozjaśni?
Ad. 3 nie podoba mi się :P
Ad. 4 Pomysł jest całkiem ok, ale pytanie  co się wydarzy jak cały przeleci? Będzie infinite animation?

**Reply by kalush81** [reply]
Ad. 2 chodzi o to aby przyciski wyborow znikaly albo byly dezaktywowane po wybraniu odpowiedzi,  aby zapobiec kilkukrotnym kliknieciom / wyborom.

**Comment by lilareliga** [comment]
opcja: wybór trudności gry np. częstotliwość odpowiedzi komputera
